build.sc

```diff
@@ -1,12 +1,6 @@
 // SPDX-License-Identifier: Apache-2.0
 
 // Build script for mill 0.6.0
-import mill._
```

Comment:
These import should not be removed.

Reply:
I put them back, I don't know how I removed them in the first place.